packages/components/src/animate/style.scss

```diff
@@ -1,6 +1,7 @@
 .components-animate__appear {
 	animation: components-animate__appear-animation 0.1s cubic-bezier(0, 0, 0.2, 1) 0s;
 	animation-fill-mode: forwards;
+	@include reduce-motion;
```

Comment:
What about the second animation of this file. just a few lines under.

Reply:
I looked at that one too, but I think it needs a more complicated fix. That animation fades in & out indefinitely to denote that something's loading or being updated. If we were to remove the animation, I think we'd need to replace it with some other sort of indicator in its place.

Reply:
(It's totally fixable, but I figured it's probably worth its own PR because it also involves a bit more complicated design problem)

Reply:
I was thinking about `.components-animate__slide-in` which is not a loading animation AFAIK

Reply:
Oh sorry — I should've looked more closely at which file that was. 😄 

I mentioned that one in the description above. It also needs some deeper thought. Since the `is-from-left` class in there moves the element off canvas, the animation is the only thing that brings it back into view. As it's written currently, if we disable the animation the sidebar will remain hidden forever. 😕

Reply:
What if you use the technique we use in e2e tests

```
* { animation-duration: 0ms !important; }
```

Reply:
Ah, that's an interesting idea! I think that's just as good as `animation: none` in this case. I'll try swapping that in and see how it goes.

Reply:
Oddly enough, that doesn't seem work reliably for me in Safari. It works great in Firefox, but in Safari, it's preventing the sidebar from appearing at all. 🤔

Reply:
🤷‍♂️ yeah, our tests only run in Chrome